<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>369</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Voluntários</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>312</width>
      <height>237</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSplitter" name="splitter_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nome"/>
       </widget>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Idade</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_idade"/>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gênero</string>
         </property>
        </widget>
        <widget class="QComboBox" name="le_genero">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Masculino</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Feminino</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Outros</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lbTurno">
         <property name="text">
          <string>Turno</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QRadioButton" name="radioMat">
          <property name="text">
           <string>Matutino</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioVesp">
          <property name="text">
           <string>Vespertino</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioNot">
          <property name="text">
           <string>Noturno</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="pb_limpar">
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pb_enviar">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_lista">
         <property name="text">
          <string>Ver Lista de Cadastrados</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
